Meadowlark-js/backends/meadowlark-mongodb-backend/src/repository/ReferenceValidation.ts

```diff
@@ -79,22 +79,25 @@ export const asUpsert = (session: ClientSession): ReplaceOptions => ({ upsert: t
 export async function validateReferences(
   documentReferences: DocumentReference[],
   descriptorReferences: DocumentReference[],
-  outRefIds: string[],
   mongoDocuments: Collection<MeadowlarkDocument>,
   session: ClientSession,
   traceId: string,
 ): Promise<string[]> {
   const failureMessages: string[] = [];
 
+  const referencedDocumentIds: string[] = documentReferences.map((dr: DocumentReference) =>
+    documentIdForDocumentReference(dr),
```

Comment:
What about descriptors?

Reply:
They are handled down around line 103. The change removing `outRefIds` didn't apply to descriptors.